churaverse-plugins-client/src/churarenPlugin/churarenCorePlugin/ui/startWindow/component/RuleExplanationWindowComponent.tsx

```diff
@@ -1,19 +1,73 @@
 import { JSXFunc } from 'churaverse-engine-client'
 import style from './RuleExplanationWindowComponent.module.scss'
+import headerImage from '../../../assets/ruleDescriptionImages/header.png'
+import itemImage from '../../../assets/ruleDescriptionImages/item_with_player.png'
+import alchemyImage from '../../../assets/ruleDescriptionImages/items_with_alchemy.png'
+import bossImage from '../../../assets/ruleDescriptionImages/attack_to_boss.png'
+import ghostImage from '../../../assets/ruleDescriptionImages/ghost_player.png'
+
 export const POPUP_GAME_START_WINDOW_BUTTON_ID = 'churaren-game-start-form-open-button'
 export const GAME_START_BUTTON = 'churaren-game-start-button'
 
 export const RuleExplanationWindowComponent: JSXFunc = () => {
   return (
-    <div className={style.container}>
-      <div id={POPUP_GAME_START_WINDOW_BUTTON_ID}>{CHURAREN_DESCRIPTION}</div>
-      <br />
-      <button className={style.popupButton} type="button" id={GAME_START_BUTTON}>
-        ちゅられん開始
-      </button>
+    <div className={style.container} id={POPUP_GAME_START_WINDOW_BUTTON_ID}>
+      <h2 className={style.title}>ちゅられん</h2>
+      <p className={style.description}>
+        ちゅられんは、みんなで協力し、制限時間内に強大なモンスターに立ち向かう協力型バトルゲーム！
+      </p>
+
+      <div className={style.section}>
+        <img src={headerImage} alt="プレイ中の様子を写した画像" className={style.headerImage} />
+      </div>
+
+      <div className={style.section}>
+        <h4>1. アイテムを集めよう！</h4>
+        <p>ボス(※1)の攻撃を掻い潜りながら、フィールドに落ちている錬金アイテムを集めます。</p>
+        <img src={itemImage} alt="アイテムとプレイヤーの写った画像" className={style.image} />
+      </div>
+
+      <div className={style.section}>
+        <h4>2. 錬金窯で攻撃を錬金！</h4>
+        <p>集めた錬金アイテムを錬金窯に投入すると、攻撃を錬金することができます。</p>
+        <img src={alchemyImage} alt="錬金窯とアイテムを持ったプレイヤーの写った画像" className={style.image} />
+      </div>
+
+      <div className={style.section}>
+        <h4>3. 錬金した攻撃でボスを撃破！</h4>
+        <p>錬金した攻撃(※2)を使って、ボスモンスターにダメージを与えましょう！</p>
+        <img src={bossImage} alt="錬金した攻撃とボスの写った画像" className={style.image} />
+      </div>
```

Comment:
[番号付きリスト](https://developer.mozilla.org/ja/docs/Web/HTML/Reference/Elements/li#%E7%95%AA%E5%8F%B7%E4%BB%98%E3%81%8D%E3%83%AA%E3%82%B9%E3%83%88)を使う方が内容の変更にも対応しやすいと思います。
また、`section`はcss内で定義されてなさそうです。



```suggestion
      <ol className={style.section}>
        <li>アイテムを集めよう！</li>
        <p>ボス(※1)の攻撃を掻い潜りながら、フィールドに落ちている錬金アイテムを集めます。</p>
        <img src={itemImage} alt="アイテムとプレイヤーの写った画像" className={style.image} />

        <li>錬金窯で攻撃を錬金！</li>
        <p>集めた錬金アイテムを錬金窯に投入すると、攻撃を錬金することができます。</p>
        <img src={alchemyImage} alt="錬金窯とアイテムを持ったプレイヤーの写った画像" className={style.image} />
        
        <li>錬金した攻撃でボスを撃破！</li>
        <p>錬金した攻撃(※2)を使って、ボスモンスターにダメージを与えましょう！</p>
        <img src={bossImage} alt="錬金した攻撃とボスの写った画像" className={style.image} />
```